### PR TITLE
Add 1.22 CI Signal shadows

### PR DESCRIPTION
**Pull request by mkorbi**
#### What type of PR is this:
/kind documentation

#### What this PR does / why we need it:
Adding CI Signal shadows to the documentation of the team

#### Which issue(s) this PR fixes:
None

#### Special notes for your reviewer:
happy to have them in the crew 👍

### CODE REVIEW

**Review by saschagrunert — APPROVED** [review]

File: releases/release-1.22/release-team.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
...
 | Enhancements | James Laverack ([@jameslaverack](https://github.com/jameslaverack) / Slack: `@james.laverack`)| Xander Grzywinski ([@salaxander](https://github.com/salaxander) / Slack: `@Xander`), Rey Lejano ([@reylejano](https://github.com/reylejano) / Slack: `@rlejano`), Grace Nguyen ([@gracenng](https://github.com/gracenng) / Slack: `@Grace Nguyen`), Supriya Premkumar ([@supriya-premkumar](https://github.com/supriya-premkumar) / Slack: `@supriya premkumar`), Joseph Sandoval ([@jrsapi](https://github.com/jrsapi) / Slack: `@Joseph`) |
-| CI Signal | Max Körbächer ([@mkorbi](https://github.com/mkorbi) / Slack: `@mkorbi`) | |
+| CI Signal | Max Körbächer ([@mkorbi](https://github.com/mkorbi) / Slack: `@mkorbi`) | [Sonia Singla](https://github.com/soniasingla) (@soniasingla / Slack: `@SONIA SINGLA`), [Hossein Salahi](https://github.com/encodeflush) (@encodeflush / Slack: `@hsalahi`), [Anna Kapuścińska](https://github.com/lambdanis) (@lambdanis / Slack: `@lambdanis`), [Rodolfo Martínez Vega](https://github.com/ramrodo) (@ramrodo / Slack: `@ramrodo`) |
```
Comment (PI-Victor) [inline releases/release-1.22/release-team.md:7]: nit: I see the rest of the file links over the github usernames, i used the same, will be a bit weird to not have all the same. e.g.: Victor Palade ([@pi-victor](https://github.com/pi-victor)

Comment (saschagrunert) [reply releases/release-1.22/release-team.md:7]: Can you open a follow-up fix for that? :)

Comment (JamesLaverack) [reply releases/release-1.22/release-team.md:7]: I made a mistake doing this too... maybe we should have a `release-team.yaml` file and generate this `.md` file with some Go?

Comment (PI-Victor) [reply releases/release-1.22/release-team.md:7]: sure, i can open a PR to fix it. 

> I made a mistake doing this too... maybe we should have a `release-team.yaml` file and generate this `.md` file with some Go?

that doesn't sound like a bad idea, who's gonna do that?

Comment (JamesLaverack) [reply releases/release-1.22/release-team.md:7]: I could raise an issue for it?

Comment (saschagrunert) [reply releases/release-1.22/release-team.md:7]: Sounds good @JamesLaverack. Let's discuss this on a dedicated issue.

**Pull request merged**